src/expressions/await-expr.md

```diff
@@ -0,0 +1,50 @@
+# Await expressions
+
+> **<sup>Syntax</sup>**\
+> _AwaitExpression_ :\
+> &nbsp;&nbsp; [_Expression_] `.` `await`
+
+Await expressions are legal only within `async` contexts, like an
```

Comment:
The reference should separately define "async contexts" (also you use backticks inconsistently, either use it everywhere or nowhere) and then refer to it.